packages/nimble-components/src/table/tests/table-column-sizing.spec.ts

```diff
@@ -564,10 +564,10 @@ describe('Table Interactive Column Sizing', () => {
                 0,
                 1
             );
-            pageObject.dragSizeColumnByRightDivider(0, [50]);
+            pageObject.dragSizeColumnByRightDivider(0, [5]);
```

Comment:
Is this change related to the rounding issues too? I think I'm missing why you had to change the values for these tests.

Reply:
Kind of. This test started failing due to the precision issue, but my workaround actually caused it (and the next test) to fail for a totally different issue. In these tests, `secondVisibleCellWidth` ends up being 133.3. It turns out that in js, 133.3 - 50 gives you 83.30000000000001. I guess previously `getCellRenderedWidth` was also returning 83.30000000000001, so the values were equal and the test passed. But now with my rounding workaround, `getCellRenderedWidth` returns 83.3, which != 83.30000000000001. By changing the delta value from 50 to 5, I avoid the numerical precision issue.

Reply:
Could we instead keep the original setup and use the jasmine toBeCloseTo: https://jasmine.github.io/api/5.1/matchers#toBeCloseTo
Sounds like it may be less finicky if there are test changes. The floating point math scenario seems like a reasonable place for it.

Reply:
eh, nvm, might require refactoring of more of the expects, can do it if it becomes flaky later